src/hooks/useAllCategories.js

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
+import { fetchCategories } from '../services/CategoriesService';
+import NoCategory from '../assets/img/CategoriesCards/NoCategory.png';
+
+const ALL_CATEGORIES_QUERY_KEY = 'categories';
+
+export const noCategoryButton = {
```

Comment:
since this is a constant, it should be named following constants code style

`NO_CATEGORY_BUTTON`